### PR TITLE
PLT-6739 Convert theme color pickers to use react-color

### PR DESCRIPTION
**Pull request by Zapix**
#### Summary
Added ColorInput component. Replaced `bootstrap-picker` with new component in `components/user_settings/custom_theme_chooser.jsx`

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/6595

#### Checklist

- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "bootstrap-colorpicker": "2.5.2",
     "chart.js": "2.7.0",
     "compass-mixins": "0.12.10",
+    "enzyme-adapter-react-15": "^1.0.1",
```
Comment (jasonblais) [inline package.json]: @Zapix Are these libraries needed for this PR?

Comment (saturninoabril) [reply package.json]: @jasonblais Yeah, I think so. This is as per enzyme recommendation.
@Zapix I suggests to fix the version like `enzyme-adapter-react-15": "1.0.1`

Comment (Zapix) [reply package.json]: @jasonblais @saturninoabril What should I do with current dependencies? Move to another PR?

Comment (jasonblais) [reply package.json]: @saturninoabril I see you submitted a separate PR which adds these dependancies: https://github.com/mattermost/mattermost-webapp/pull/111

Should that PR be merged first, and then @Zapix can help rebase his branch afterwards?

Comment (saturninoabril) [reply package.json]: @Zapix Just update this PR with fix version like `enzyme-adapter-react-15": "1.0.1`. That is without `^`.
@jasonblais Whichever came first. The last one will do the rebase.

Comment (Zapix) [reply package.json]: I"ve updated pr please check

**Pull request merged**